ui/src/time_picker.tsx

```diff
@@ -79,8 +79,11 @@ const
     toolbarLabel: { maxWidth: '70%' }
   }),
   popoverProps: Partial<PopperProps> | undefined = {
+    container: () => document.querySelector(".ms-Panel") || document.body,
     placement: 'bottom-start',
+    style: { pointerEvents: 'auto' },
```

Comment:
What is this for?

Reply:
If the container element is Panel element which uses `pointer-events: 'none'` we need to override this to make the time picker popup responsive to clicks. Let's see if we could use `XTimePicker` as a container.

Reply:
Got it, makes sense.